docs/sdk/pubsub/fragments/ios/getting-started.md

```diff
@@ -17,15 +17,15 @@ When used with `AWSIoTDataManager`, PubSub is capable of signing request accordi
 The `Podfile` that you configure to install the AWS Mobile SDK must contain the `AWSIoT` pod:
 
 ```ruby
-    platform :ios, '9.0'
+platform :ios, '9.0'
 
-    target :'YOUR-APP-NAME' do
-      use_frameworks!
+target :'YOUR-APP-NAME' do
+  use_frameworks!
 
-        pod  'AWSIoT', '~> 2.13.0'
-        # other pods
+  pod  'AWSIoT', '~> 2.13.0'
```

Comment:
In our IoT [example](https://github.com/awslabs/aws-sdk-ios-samples/blob/master/IoT-Sample/Swift/Podfile), we also include AWSMobileClient.  I wonder if we should also call out AWSMobileClient?